include/functions.h

```diff
@@ -918,7 +918,7 @@ void func_800BD888(Actor*, struct_800BD888_arg1*, s16, s16); // This function is
 // void func_800BDB6C(void);
 void func_800BDC5C(SkelAnime* skelAnime, ActorAnimationEntry animation[], s16 index);
 // void func_800BDCF4(void);
-void func_800BDFB0(void);
+void func_800BDFB0(Actor* thisx, GlobalContext* globalCtx);
```

Comment:
```suggestion
void func_800BDFB0(Actor* actor, GlobalContext* globalCtx);
```
Nearly all cases I've seen of `Actor*` in `functions.h` is labelled `actor` instead of `thisx`. This is probably a continued discussion outside the scope of this PR as I'm sure either is fine , but using `actor` would be more consistent with what's already here

Reply:
Yeah, that's fair enough.

Reply:
I agree with EllipticEllipsis on this one. Since it is explicitly used as an `ActorFunc` by some actor, it makes more sense to have `thisx` instead of `actor`.

Reply:
Switched to `actor`. Also named the function `Actor_Noop` to match OoT.